annotations/templates/annotations/settings.html

```diff
@@ -1,32 +1,71 @@
 {% extends "annotations/base.html" %}
 {% load staticfiles %}
-{% load i18n admin_static %}
-{% load crispy_forms_tags %}
-
 {% block main %}
-<div id="main">
-
-		<div class="col-sm-8 col-sm-offset-2">
-			<form action="#" method="POST">
-				<div id="settings" class="panel panel-default">
-					<div class="panel-heading">Update your profile</div>
-					<div class="panel-body">
-						<!-- <div class="well"> -->
-
-						{% crispy form %}
+<script type="text/javascript">
+/*
+   Function called when file input updated. If there is a file selected, then
+   start upload procedure by asking for a signed request from the app.
+*/
+function init_upload(){
+    var files = document.getElementById("file_input").files;
+    var file = files[0];
+    if(file == null){
+        alert("No file selected.");
+        return;
+    }
+    get_signed_request(file);
+}
+/*
+   Bind listeners when the page loads.
+*/
+(function() {
+    document.getElementById("file_input").onchange = init_upload;
+})();
 
-						<!-- </div> -->
-					</div>
-
-					<div class="panel-footer clearfix">
-						<span class="pull-right">
-							<input type="submit" class="btn btn-primary" value="Submit" />
-						</span>
-					</div>
-
-
-				</div>
-			</form>
-		</div>
+</script>
+<div id="main">
+   <div class="col-sm-8 col-sm-offset-2">
+      <form action="#" method="POST" enctype="multipart/form-data">
+         <div id="settings" class="panel panel-default">
+            <div class="panel-heading">Update your profile</div>
+            <div class="panel-body">
+      <form  action="#" method="post"  enctype="multipart/form-data">
+      {% csrf_token %}
+      <div class="form-group">
+      <label for="fullname">Full Name</label>
+      <input type="text" class="form-control" id="id_fullname" value="{{ full_name }}">
+      </div>
+      <div class="form-group">
+      <label for="email">Email Address</label>
+      <input type="email" class="form-control" id="id_email" value="{{ email }}">
+      </div>
+      <div class="form-group">
+      <label for="affiliation">Affiliation</label>
+      <input type="text" class="form-control" id="id_affiliation" value="{{ affiliation }}">
+      </div>
+      <div class="form-group">
+      <label for="location">Location</label>
+      <input type="text" class="form-control" id="id_location" value="{{ location }}">
+      </div>
+      <div class="form-group">
+      <label for="link">Link</label>
+      <input type="text" class="form-control" id="id_link" value="{{ link }}">
+      </div>
+      <div class="form-group">
+      <label for="imagefile">Please upload an image</label>
+      <input type="file" id="file_input"/>
```

Comment:
If this is inside the ```<form></form>```, then the file will be uploaded to VogonWeb, too. We don't want that. See the first code-cell in this section: https://devcenter.heroku.com/articles/s3-upload-python#setting-up-the-client-side-code